Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM golang:alpine
```

Comment:
also, do a little research on which distro will be based for video processing and supports GPU acceleration.

Reply:
I found these resources for GPU accelerated containers:
https://learn.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/gpu-acceleration#run-a-container-with-gpu-acceleration
https://unrealcontainers.com/docs/concepts/gpu-acceleration#nvidia-gpu-support-for-linux-containers

Reply:
https://github.com/akofman/docker-video-processing